src/pages/home/base/mobile-core/release-notes.md

```diff
@@ -8,6 +8,13 @@ keywords:
 
 # Release Notes
```

Comment:
Actually, nothing to do with yours, but something SUPER minor that's bugging me that I just noticed - is there a reason that for this one it's "Release Notes" but on the homepage it's "Release notes"?

I _believe_ Adobe style guides prefer sentence casing for titles right now, so if we can change this to "Release notes" before you merge, that would be great. Thanks!